resources/lang/en/tournament.php

```diff
@@ -33,7 +33,7 @@
             'cancel' => 'Cancel Registration',
         ],
         'entered' => 'Your are registered for this tournament.<br><br>Note that this does not mean you have been assigned to a team.<br><br>Further instructions will be sent to you via email closer to the tournament date, so please ensure your osu! account\'s email address is valid!',
-        'login_to_register' => 'Please :login to view registration details!',
+        'login_to_register' => 'Please sign in to view registration details!',
```

Comment:
no need to change this one. The text uses whatever `users.anonymous.login_link` is.

Don't forget to remove comment for other locales.